### ISSUE TITLE
train the model

### ISSUE THREAD

**Issue by HoracceFeng** [hellochick/ICNet-tensorflow#2]
Hello,

I am interested in your work and would like to try using your code to train my dataset. Would you provide the training code and give me some suggestions about this?

Thank you so much.

**Comment by hellochick** [comment]
Hey @HoracceFeng , I will provide the training code in recent days. What kinds of dataset do you want to use? Maybe I can give you some suggestions.

**Comment by twinsyssy1018** [comment]
waiting for the train code

**Comment by HoracceFeng** [comment]
Hi, the datasets are some gray level images, and just have two classes.

**Comment by ifangcheng** [comment]
waiting for the train code +1
And I just want to do perdestrian segmentation.
Because after I compared the provided checkpoint  icnet model with fcn-8s  in terms of perdestrian segmentation accuracy, it seems that the icnet model here do not show advantage and perform even worse than fcn_8s. 
So, maybe i should train a   specialized icnet model for perdestrian segmentation?

**Comment by hellochick** [comment]
Hey, @ifangcheng , I think ICNet model focus on "real-time" segmentation, so it will get faster inference time at a sacrifice of accuracy. So, if you just want to do pedestrian segmentation, you can take PSPNet into your consideration, much slower but accurate

**Comment by ifangcheng** [comment]
@hellochick， thanks for the suggestion.
However, real-time is also very important for me.
I suppose ICnet when trained specially for perdestrian segmentation may improve the accuracy than the provided model which is not focusing on perdestrian segmentation,  am I right?
does that make sense?

**Comment by UndeadBlow** [comment]
Hi, @hellochick, any updates about train?

**Comment by hellochick** [comment]
Hey, @UndeadBlow , I have already done, now I am checking the performance.

**Comment by hellochick** [comment]
Hey guys, I have updated the repo.

**Comment by HoracceFeng** [comment]
Thank you so much!

Sent from my iPhone

On 15 Nov 2017, at 20:18, CodingChick <notifications@github.com<mailto:notifications@github.com>> wrote:


Hey guys, I have updated the repo.
